### PR TITLE
Update doc to clarify OpenDevin mission and link directly to content

### PR DESCRIPTION
**Pull request by neubig**
**What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**

This PR does two things:
1. It updates our doc to clarify our mission (not to replicate Devin, but be a community-based open source platform for software development)
2. It changes the link on the README to go directly to the contentful doc page, instead of the landing page

**Other references**

[Current doc page](https://opendevin.github.io/OpenDevin/) for reference

### CODE REVIEW

File: docs/src/pages/faq.tsx [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
   return (
     <>
       <Layout title="FAQ" description="Frequently Asked Questions">
         <div id="faq" className="faq-container">
           <div className="faq-title">Frequently Asked Questions</div>
+          <div className="faq-section">
+            <div className="faq-section-title">What is OpenDevin?</div>
+            <p>
+              <span className="highlight">OpenDevin</span>{" "}
+              is an autonomous software engineer that can solve solve software engineering
```
Comment (tobitege) [inline docs/src/pages/faq.tsx:15]: duplicate "solve"

File: docs/src/pages/faq.tsx [hunk 2]
```diff
@@ -71,6 +58,27 @@ export default function FAQ() {
               where USERNAME is your GitHub username.
             </div>
           </div>
+          <div className="faq-section">
+            <div className="faq-section-title">How is OpenDevin Different from Devin?</div>
```
Comment (tobitege) [inline docs/src/pages/faq.tsx:62]: lowercase "different"

File: docs/src/pages/faq.tsx [hunk 3]
```diff
@@ -71,6 +58,27 @@ export default function FAQ() {
...
+              inspiration for OpenDevin. They both aim to do a good job at solving software
+              engineering tasks, but OpenDevin you can download, use, and modify, while Devin
+              you can only use through the Cognition site. In addition, OpenDevin has evolved
+              beyond the initial inspiration, and now serves as a community-driven ecosystem for
+              agent development in general, and we'd love to have you join and
+              <a href="https://github.com/OpenDevin/OpenDevin/blob/main/CONTRIBUTING.md">contribute</a>!
+            </p>
+          </div>
+          <div className="faq-section">
+            <div className="faq-section-title">How is OpenDevin Different from ChatGPT?</div>
```
Comment (tobitege) [inline docs/src/pages/faq.tsx:75]: lowercase "different"

**Review by tobitege — CHANGES_REQUESTED** [review]
Overall good, little nits.

**Comment by neubig** [comment]
Thanks @tobitege , fixed!

**Review by tobitege — APPROVED** [review]
Great, good to go 😃

**Pull request merged**